src/infra/services/loggerImpl.service.ts

```
@@ -0,0 +1,101 @@
+
+export class LoggerServiceImpl implements LoggerService {
+  constructor(
+    private readonly httpClient: HttpClient,
+    private readonly baseUrl: string,
+    private readonly apiKey: string,
+  ) {}
+
+  // buffer to store logs temporarily
+  private _logsBuffer: LoggerService.Buffer[] = [];
+
+  private _labels: Object = {};
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)